src/gp_dag.hpp

```diff
@@ -44,9 +47,23 @@ class GPDAG {
   // Get the index of a PLV of a given type and with a given index.
   static size_t GetPLVIndexStatic(PLVType plv_type, size_t node_count, size_t src_idx);
   size_t GetPLVIndex(PLVType plv_type, size_t src_idx) const;
+  
+  size_t GetGPCSPIndex(const Bitset &pcsp) const {
```

Comment:
Could we rename this "GetGPCSPIndexWithDefault" to clearly signal that it has special behavior?

Also, can we move this method and the method below into a cpp file?

Reply:
Done.